### PR TITLE
Add a fallback resource when compositions fail to load

### PR DESCRIPTION
**Pull request by gpeal**
This may happen either with invalid inputs or when network loading fails.

### CODE REVIEW

**Comment by gpeal** [comment]
**Snapshot Tests**
**28**: [Report](https://happo.io/a/27/report/0c9877d35131558cdc62de5c119fc52cb90a3c43-android28) [Diff](https://happo.io/a/27/p/27/compare/master-android28/0c9877d35131558cdc62de5c119fc52cb90a3c43-android28)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+# 3.1.0
+### Features and Improvements
+* **Breaking Change** Replace JsonReader parsing APIs with InputStream variants to prevent 
+exposing Lottie's copy of Moshi's json parser.
+* Add the ability to catch all Lottie composition errors with `setFailureListener` and 
+`resetFailureListener` (#1321).
+* Add the ability to set a fallback drawable res when Lottie failes to parse a composition or
```
Comment (digitalbuddha) [inline CHANGELOG.md]: resource.... fails

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+# 3.1.0
+### Features and Improvements
+* **Breaking Change** Replace JsonReader parsing APIs with InputStream variants to prevent 
+exposing Lottie's copy of Moshi's json parser.
+* Add the ability to catch all Lottie composition errors with `setFailureListener` and 
+`resetFailureListener` (#1321).
+* Add the ability to set a fallback drawable res when Lottie failes to parse a composition or 
+load it from the internet with `setFallbackResource` and the `setFallbackRes` xml attribute.
```
Comment (digitalbuddha) [inline CHANGELOG.md]: from network

**Comment by gpeal** [comment]
Thanks for taking a look @fawaad!

**Pull request merged**